tests/test_transforms.py

```diff
@@ -525,11 +523,11 @@ def testFromMatches(self):
         initialIwcToSky = lsst.afw.geom.getIntermediateWorldCoordsToSky(initialWcs)
         for i in range(nPoints):
             refRec = ref.addNew()
-            skyPos = lsst.afw.geom.Point2D(
+            raDeg, decDeg = (
                 np.random.uniform(low=bbox.getMinX(), high=bbox.getMaxX()),
-                np.random.uniform(low=bbox.getMinY(), high=bbox.getMaxY())
+                np.random.uniform(low=bbox.getMinY(), high=bbox.getMaxY()),
```

Comment:
Is this trailing comma needed?

Reply:
It is not necessary but it is good Python style.